docs/design-language.md

```diff
@@ -223,6 +223,22 @@ A few guidelines should be followed to ensure that the list element looks unifor
 
 When a hover state is triggered, it cannot just rely on a colour change. There also must be a state change that shows the differentiation. For example, in our programs, there is a background colour change, as well as the addition of a side border colour. Hover states always include a styling change, and the background colour will get a gradient.
 
+## Loading
+
+Two options are available to denote a loading state, the "3 dot loader" and "Skeleton". Skeleton is the preferred method with the 3 dots being phased out where applicable.
+
+### Skeleton
+
+This loader can be customized to display a placeholder of incoming content. This helps improve perceived responsiveness. When using this method it is best to create a simplified layout, just enough to give the user an idea of whats coming. Animate skeleton using the "wave" style. Read more about skeleton in the **<a href="https://mui.com/material-ui/react-skeleton/" target="_blank">MUI docs</a>.**
+
+<img src="/img/loaders/skeleton-loader.gif" alt="3 dot loader animation" title="3 dot loader animation" className="img-example" />
```

Comment:
Love the the example of a skeleton. Can we add another one or two to show various layouts of a skeleton loader?

Reply:
Updated.

Reply:
We should add in more realistic example GIFs with more of the UI showing it in action. This should be done for both the GIF examples for the skeleton and the three dot loader.

Reply:
Added.